CONTRIBUTING.md

```diff
@@ -1,3 +1,4 @@
+# Contributing 
```

Comment:
Thanks for the fix up! Please keep whitespace consistent though, and add a blank line between the title on the text below it.

Reply:
Done.